templates/puppetca_puppet_cert.yml.erb

```diff
@@ -0,0 +1,4 @@
+---
+:ssldir: <%= scope.lookupvar("foreman_proxy::ssldir") %>
+#:puppetca_use_sudo: true
```

Comment:
Does having these parameters here commented out serve any purpose?  If they're worth including here, they should probably also go in `puppetca.yml.erb` when split configs is false.